quickstarts/microsoft.documentdb/cosmosdb-mongodb-autoscale/main.bicep

```diff
@@ -141,24 +152,14 @@ resource collection1 'Microsoft.DocumentDb/databaseAccounts/mongodbDatabases/col
         {
           key: {
             keys: [
-              'user_id'
-              'user_address'
+              'product_name'
+              'product_category_name'
             ]
           }
           options: {
             unique: true
           }
         }
-        {
-          key: {
-            keys: [
-              '_ts'
-            ]
-            options: {
-              expireAfterSeconds: 2629746
-            }
-          }
-        }
       ]
       options: {
         'If-Match': '<ETag>'
```

Comment:
The ADO build is raising a warning here:

```
##[error]D:\a\1\s\quickstarts\microsoft.documentdb\cosmosdb-mongodb-autoscale\main.bicep(164,7) : Warning BCP037: The property "options" is not allowed on objects of type "MongoDBCollectionResourceOrMongoDBCollectionGetPropertiesResource". Permissible properties include "analyticalStorageTtl". If this is an inaccuracy in the documentation, please report it to the Bicep Team. [https://aka.ms/bicep-type-issues]
```

`properties.options` does not have an `If-Match` property defined, so this may need to be corrected in the Swagger.